lang/fr-on.js

```diff
@@ -0,0 +1,123 @@
+/* eslint quotes: 0 */
```

Comment:
Random side note, but I wonder if we added `fr-on.js` to our `.gitignore` if the file just wouldn't be committed by the automation 🤔

Reply:
I think it's worth trying.  🤔